src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/adjourncase/AdjournCaseAboutToSubmitHandler.java

```diff
@@ -79,6 +84,32 @@ public PreSubmitCallbackResponse<SscsCaseData> handle(CallbackType callbackType,
             sscsCaseData.getAdjournment().setGeneratedDate(LocalDate.now());
         }
 
+        updateHearingChannel(sscsCaseData);
+
         return new PreSubmitCallbackResponse<>(sscsCaseData);
     }
+
+    private static void updateHearingChannel(SscsCaseData sscsCaseData) {
+
+        if (sscsCaseData.getAdjournment().getTypeOfNextHearing() != null) {
```

Comment:
The whole point of adding adjournmentInProgress was to avoid reliance on certain fields in Adjournment not being null. 
```suggestion
        if (isYes(sscsCaseData.getAdjournment().getAdjournmentInProgress() && sscsCaseData.getAdjournment().getTypeOfNextHearing() != null)) {
```

Reply:
Please also cover all scenarios with tests, ie 
in progress && type != null; 
not in progress && type != null; 
in progress && type == null; 
not in progress && type == null

Reply:
Having said all of this, I am now doubting whether I am right about this. Remembering Pankaj's case - this code exists in GAPS as well and therefore should not be partitioned off with our new adjournment? Is that correct? I'm not entirely sure now, but lets have a chat about it when you are in.

Reply:
As discussed, this whole PR can (probably) be removed. All functionality will be run on the hearings side.

Reply:
Removed the code.

Reply:
To clarify, my previous statement turned out to be wrong. We do require this PR on the tribunals side.